packages/data/src/reviews/test/reducer.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Just a little thing. we could consider using the jest environment tag here to run these tests in a `node` environment. For every suite we run in `node` we do get a upfront performance bonus because setting up a jsdom env happens per suite and I've seen some metrics saying this can add an additional 1 or more seconds per suite, which will add up over time as we add significantly more tests. Any reducers, generators, anything that doesn't interact with the DOM could be run under the node environment.

The tag is like:

```
/**
 * @jest-environment node
 */
```

Reply:
Sounds like a good idea. I'm getting `Invalid JSDoc tag name "jest-environment".eslint(jsdoc/check-tag-names)` linter error. May need to address this in a follow up.